docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     container_name: dsp_nginx
     volumes:
       - ./nginx/nginx.conf:/etc/nginx/nginx.conf
-      - ./nginx/config:/etc/ssl
+      - /opt/certs:/etc/ssl
```

Comment:
This will make cause problems with local deployments, but that is alright because we have docker-compose-dev

Reply:
Yes, use the dev files for dev deployments.